examples/advanced_behaviors/index.xml.njk

```diff
@@ -1,3 +1,8 @@
+---
+title: "Advanced Behaviors"
+permalink: "/advanced_behaviors/index.xml"
+tags: root
+---
```

Comment:
Instead of renaming the xml files, could we just include them? That would allow us to continue linting the XML markup

```suggestion
---

{% include 'examples/advanced_behaviors/index.xml' %}
```

Reply:
But the xml files may no longer be valid, since they will contain template syntax.

A better approach may be to run linting/validation on the generated XML files. We could check those into the repo.

Reply:
Can templates have the `.xml` extension, or does the `.xml.njk` extension is required for parsing to work?

Reply:
Linting post-processing can become a headache depending on how complex templates becomes, particularly when nesting multiple template statements. That might be a non problem for this project but just wanted to raise this.

Reply:
> Can templates have the `.xml` extension, or does the `.xml.njk` extension is required for parsing to work?

The eleventy project detects which template handler to used based on the extension, so that's why `.njk` is included. I looked into it, it's possible to add a new extension, but I'm not sure it's worth it.

Regarding linting, it maybe matters less if we're not writing a lot of boilerplate.